docs/REST_API-Draft/draft.md

```diff
@@ -183,16 +216,18 @@ Parametry:
 | Nazwa | Wymagany | Opis | Domyślnie |
 |-------|----------|------|-----------|
 | measurement-names | opcjonalny | nazwa poszukiwanych pomiarów | |
```

Comment:
chyba wystarczy liczba pojedyncza (measurement-name) (tak samo w host-names), bo użytkownik i tak w GUI nie poda raczej kilku (nie ma na to wymagania)